framework/set/provisioning/setConfigTF.go

```diff
@@ -4,34 +4,39 @@ import (
 	"os"
 	"strings"
 
+	"github.com/rancher/shepherd/clients/rancher"
 	framework "github.com/rancher/shepherd/pkg/config"
 	"github.com/rancher/tfp-automation/config"
 	"github.com/sirupsen/logrus"
 )
 
 const (
-	aks         = "aks"
-	eks         = "eks"
-	gke         = "gke"
-	azure_rke1  = "azure_rke1"
-	azure_rke2  = "azure_rke2"
-	azure_k3s   = "azure_k3s"
-	ec2RKE1     = "ec2_rke1"
-	ec2RKE2     = "ec2_rke2"
-	ec2K3s      = "ec2_k3s"
-	linodeRKE1  = "linode_rke1"
-	linodeRKE2  = "linode_rke2"
-	linodeK3s   = "linode_k3s"
-	vsphereRKE1 = "vsphere_rke1"
-	vsphereRKE2 = "vsphere_rke2"
-	vsphereK3s  = "vsphere_k3s"
-	rke1        = "rke1"
-	rke2        = "rke2"
-	k3s         = "k3s"
+	aks             = "aks"
+	rancherBaseline = "rancher-baseline"
```

Comment:
nit - can we have this as the last thing in the const list? Just because we have the clusters and then the modules. So let's have this PSACT come after both of those.